data_steward/validation/main.py

```diff
@@ -1096,7 +1096,7 @@ def run_retraction_cron():
         cleaning_args = [
             '-p', project_id, '-d', dataset_id, '-b', sandbox_dataset_id,
             '--data_stage', CRON_RETRACTION, '--run_as',
-            f'{project_id}@appspot.gserviceaccount.com', '-s'
+            f'{project_id}@appspot.gserviceaccount.com'
```

Comment:
Do we need to remove `'-s'` here? I think it is still helpful to have the console log, if this `'-s'` is irrelevant from the local log file creation.

Reply:
Yeah, the log file was my concern too. Should I keep it? I just don't want us to deploy again.